### DISCUSSION TITLE
v1.0.0-beta

### DISCUSSION THREAD

**Discussion by michigang1** [michigang1/michat#3] [Announcements]
**Full Changelog**: https://github.com/michigang1/michat/commits/main 

- First beta-version of application - **Minimum Viable Product** (MVP)
    - there are only basic 2 functions: authentication and chatting

<hr /><em>This discussion was created from the release <a href='https://github.com/michigang1/michat/releases/tag/v1.0.0-beta'>v1.0.0-beta</a>.</em>

**Comment by Snare295** [comment]
https://michigang1.github.io/michat/ not showing chat. Or if it's apk, can you include 😫

**Reply by michigang1** [reply]
it's readme on GH pages

**Reply by Snare295** [reply]
> it's readme on GH pages
> 
> 
> 

So can you publish apk or other build?

**Reply by michigang1** [reply]
in the current version - no(

**Reply by michigang1** [reply]
i have some bugs
